### PR TITLE
Add YukiCoder Budge

### PR DESCRIPTION
**Pull request by ganyariya**
Hello, @kehsihba19 !
I am ganariya, a student in Japan.

When I tried to make competitive-programming badges, I found your repository!
And I like to add [yukicoder](https://yukicoder.me/), as a sub-selection.
(YukiCoder does not have color system, so I fix color as green, theme color of yukicoder ...)

Could you give me a review and opinion :) ?

Thank you very much!

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,50 +1,64 @@
...
+
+
+@app.route("/yukicoder/<user>")
+def yukicoder(user):
+    x = get_yuki(user)
+    level, colour = x[0], x[1]
+    badge = pybadges.badge(left_text='YukiCoder', right_text=level, right_color=colour)
+    response = flask.make_response(badge)
+    response.content_type = 'image/svg+xml'
+    return response
```
Comment (ganyariya) [inline main.py:54]: add flask route about yukicoder

File: data.py [hunk 2]
```diff
@@ -1,78 +1,92 @@
...
+        col = '#FF0000'
+    return [val, col]
+
+
+def get_yuki(user):
+    url = f'https://yukicoder.me/api/v1/user/name/{user}'
+    json_data = requests.get(url).json()
+    level = str(json_data['Level'])
+    color = '#2ecc71'
+    return [level, color]
```
Comment (ganyariya) [inline data.py:92]: add yukicoder query
(yukicoder has api-endpoint.)

File: data.py [hunk 3]
```diff
@@ -1,78 +1,92 @@
...
-    	col="#1427B2"
-    elif(y>1899 and y<=2199):
-    	col="#700CB0"
-    elif(y>2199 and y<=2399):
-    	col="#F9A908"
+    r = requests.get(f"https://codeforces.com/profile/{user}").text
+    soup = bs(r, 'lxml')
+    s = soup.find('span', class_='smaller')
+    s = s.text
+    rating = (re.findall(r'\d+', s)[0])
```
Comment (ganyariya) [inline data.py:13]: Sorry, I made lint of python, pep8... so large line changed...

**Comment by kehsihba19** [comment]
Thank you for adding the support of Yukicoder OJ.
I will be happy to merge the pull request and add you as an active contributor to the repo.
Thanks again.
Good Work :)

**Pull request merged**